### DISCUSSION TITLE
How to skip a file?

### DISCUSSION THREAD

**Discussion by ethanforvest** [DIGITALCRIMINALS/OnlyFans#328] [Q&A]
There is a model that has a huge size of video that I don't want to download. Is there a way to skip that file? thank you 

edit: 
** I'm sorry for the duplicate, I didn't mean to, I can't see an option to delete the discussion. what should I do now? again, i'm really sorry!!

**Comment by DIGITALCRIMINAL** [comment]
Add the model's name in the blacklist

**Reply by ethanforvest** [reply]
Thanks for the reply! I meant I don't want to completely block the model. I want to skip a specific video from her page. (Still, I want to download her other stuff, but not the video) Hoping you're getting what I'm trying to say.
